custodia/secrets.py

```diff
@@ -23,6 +23,8 @@ def __init__(self, config, section):
 
     def _db_key(self, trail):
         if len(trail) < 2:
+            self.logger.debug('Forbidden action: Operation only permitted '
```

Comment:
Style nit pick. You can wrap the lone line a bit different to make the text more readable:

```
self.logger.debug(
   'Forbidden action: ...')
```